src/services/internal/didcomm_encrypted.rs

```diff
@@ -95,10 +95,17 @@ impl DIDCommEncryptedService {
             }
         };
 
-        let mut message = Message::new()
+        let mut message = match Message::new()
             .from(&my_did)
             .to(&[to_did])
-            .body(&body.to_string());
+            .body(&body.to_string())
+        {
+            Ok(v) => v,
+            Err(e) => {
+                log::error!("Failed to initialize message. with error = {:?}", e);
```

Comment:
```suggestion
                log::error!("Failed to initialize message with error = {:?}", e);
```

The .  should be remove for perfect sentence!

Reply:
Thank you!
https://github.com/nodecross/nodex/pull/213/commits/536586d93e89762d777cf771159bb883c0096e11